src/ControllerStatusIO.h

```diff
@@ -8,6 +8,9 @@
 #ifndef MOTOROS2_CONTROLLER_STATUS_IO_H
 #define MOTOROS2_CONTROLLER_STATUS_IO_H
 
+#include "MotoPlusExterns.h" // for MAX_ERROR_COUNT
```

Comment:
It would be more consistent (style-wise) to remove this line and bump [the include](https://github.com/Yaskawa-Global/motoros2/blob/c2453273e80dfe2841d5338b08d489af943c5214/src/MotoROS.h#L98) up above [this line](https://github.com/Yaskawa-Global/motoros2/blob/c2453273e80dfe2841d5338b08d489af943c5214/src/MotoROS.h#L86).

Reply:
Probably.

It also highlights the fragility of the current style of `#include`-ing MotoROS2 currently uses.

We should really switch to `include-what-you-use` instead.

Reply:
I've re-ordered the `#include`s, but instead of squashing it into e8e66c77073e6fdf98e295b24446fb0e9a73657d, I've kept it as a separate commit.

Just to have a clear reminder.